config/settings/development.yml

```diff
@@ -2,9 +2,9 @@ host_inventory_url: http://inventory-web:8081
 remediations_url: https://ci.cloud.redhat.com
 rbac_url: https://ci.cloud.redhat.com
 # disable_rbac: true
-# platform_basic_auth:
-#   - username
-#   - password
+platform_basic_auth:
+  username:
+  password:
```

Comment:
This should be commented out by default. Also, all the currently commented out settings are examples, so it'd be good to match that with something like:

```
# platform_basic_auth:
#   username: myuser
#   password: changeme
```

Reply:
Sure I can comment it out. There is no difference. I believe that the above is self-explanatory.